DataTransferObject/Form/BaseUser.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\DataTransferObject\Form;
+
+use SumoCoders\FrameworkMultiUserBundle\User\User as UserEntity;
+
+class BaseUser extends User
+{
+    /**
+     * @var string
+     */
+    public $password = '';
+
+    /**
+     * @param UserEntity $user
+     *
+     * @return User
+     */
+    public static function fromUser(UserEntity $user)
+    {
+        $dataTransferObject = new self();
```

Comment:
een naam die duidelijk maakt waarvan het een dataTransferObject is,

dit is het zelfde als een database record van een user terug krijgen en het gewoon record noemen